.github/workflows/main.yml

```diff
@@ -129,29 +123,27 @@ jobs:
           name: "endlesskeyapp.unsigned.appx"
           path: "endlesskeyapp.unsigned.appx"
 
-      - name: Sign artifact for release
-        id: sign_for_release
-        if: startsWith(github.ref, 'refs/tags/') && github.event.inputs.sign_test_build == false
+      - name: Sign artifact for testing
+        id: sign_for_testing
+        if: startsWith(github.ref, 'refs/tags/') == false
         shell: pwsh
         env:
-          SIGNPATH_ORG_ID: ${{ secrets.SIGNPATH_ORGANIZATION_ID }}
-          SIGNPATH_USER_TOKEN: ${{ secrets.SIGNPATH_API_TOKEN }}
+          SIGN_PHRASE: ${{ secrets.SELFSIGN_PHRASE }}
         run: |
-          Set-PSRepository PSGallery -InstallationPolicy Trusted
-          Install-Module -name SignPath
-          Submit-SigningRequest `
-            -CIUserToken $env:SIGNPATH_USER_TOKEN `
-            -OrganizationId $env:SIGNPATH_ORG_ID `
-            -ProjectSlug Endless_Apps `
-            -SigningPolicySlug Endless_Apps_Signing_for_Microsoft_Store `
-            -ArtifactConfigurationSlug Endless_Key_APPX_Package_for_Microsoft_Store `
-            -InputArtifactPath endlesskeyapp.unsigned.appx `
-            -OutputArtifactPath endlesskeyapp.appx `
-            -WaitForCompletion
+          & "${Env:ProgramFiles(x86)}\Windows Kits\10\App Certification Kit\signtool.exe" `
+            sign /fd SHA256 /a /f .buildkite_win/selfsign.pfx /p $env:SIGN_PHRASE endlesskeyapp.unsigned.appx
+          mv endlesskeyapp.unsigned.appx endlesskeyapp.testsigned.appx
```

Comment:
Let's drop the `.unsigned` part from the unsigned package (on the "Packaging" step a few lines above), since this is what we will release in the end (an unsigned package). Then maybe just call `.selfsigned` here instead of `.testsigned`?

Also, I think we should drop `app` from the name and just call it `endlesskey.appx`, but let's leave that part for a separate commit.

Reply:
ok